packages/go_router/test/src/custom_transition_page_test.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:flutter/material.dart';
```

Comment:
@Ascenio You need to include the license header at the top of your file 

refer this 

https://github.com/flutter/packages/blob/2ca61c1c0aba93a8b7d045ce4bcae8d5578dcfd9/packages/go_router/test/go_router_test.dart#L1-L4